server/plugin/virtual_agent.go

```diff
@@ -256,12 +267,84 @@ func (p *Plugin) ProcessResponse(data []byte) error {
 			if _, err = p.DMWithAttachments(userID, p.CreateOutputCardAttachment(&data)); err != nil {
 				return err
 			}
+		case *OutputImage:
+			post, err := p.CreateOutputImagePost(res, userID)
+			if err != nil {
+				return err
+			}
+
+			if _, err = p.dm(userID, post); err != nil {
+				return err
+			}
 		}
 	}
 
 	return nil
 }
 
+func (p *Plugin) CreateOutputImagePost(body *OutputImage, userID string) (*model.Post, error) {
+	channel, appErr := p.API.GetDirectChannel(userID, p.botUserID)
+	if appErr != nil {
+		p.API.LogInfo("Couldn't get bot's DM channel", "user_id", userID, "error", appErr.Message)
```

Comment:
```suggestion
		p.API.LogInfo("Couldn't get bot's DM channel", "UserID", userID, "Error", appErr.Message)
```
Why `LogInfo` here?
And are we using snake_case everywhere for keys in the logs?